### PR TITLE
Fixed docker compose (consul service)

### PR DESCRIPTION
**Pull request by turric4n**
**What this PR does / why we need it**:

Fix docker-compose up

**Which issue(s) this PR fixes**:

Any

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

Any

Please make sure you've completed the relevant tasks for this PR, out of the following list:

- [V ] Code compiles correctly
- [V] Unit tests passing
- [V] End-to-end tests passing

### CODE REVIEW

**Comment by sungam3r** [comment]
What problem does this PR fix?

**Comment by sungam3r** [comment]
I'm not an expert in docker and I've never used anything related to docker from that repo. I don't know the purpose of that file.

**Comment by sungam3r** [comment]
Also there is no CI run triggered.

**Comment by turric4n** [comment]
Current docker-compose file doesn't work because consul latest image not exists. I fixed that. That should be merged asap. Thanks.

**Comment by turric4n** [comment]
> I'm not an expert in docker and I've never used anything related to docker from that repo. I don't know the purpose of that file.

That's ok. A Docker Compose file simplifies the orchestration of Docker services. 

Here we use Docker Compose to easily set up services so that tests can utilize these services locally.

File: docker-compose.yml [hunk 1]
```diff
@@ -134,7 +134,7 @@ services:
         - "${TMPDIR:-/tmp/localstack}:/tmp/localstack"
         - "/var/run/docker.sock:/var/run/docker.sock"
     consul:
-        image: consul:latest
+        image: consul:1.15.4
```
Comment (rob-baldwin) [inline docker-compose.yml:137]:
```suggestion
        image: hashicorp/consul:latest
```

We should be using the updated supported image, not the deprecated image.

Comment (turric4n) [reply docker-compose.yml]: You are right! Thanks!

Comment (sungam3r) [reply docker-compose.yml]: Ah, I fixed that in #1853 for consul CI. OK.

**Pull request merged**